WordPress/Jetpack/Classes/ViewRelated/WordPress-to-Jetpack Migration/Common/MigrationAppDetection.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+
+enum WordPressInstallationState {
+    case wordPressNotInstalled
+    case wordPressInstalledNotMigratable
+    case wordPressInstalledAndMigratable
+}
+
+struct MigrationAppDetection {
```

Comment:
I'm open to changing where we store this and what it's named, I kept this local to the migration code.